eth2near/eth2near-block-relay-rs/Cargo.toml

```
@@ -4,17 +4,20 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-log = { version = "0.4", features = ["std", "serde"] }
 types =  { git = "https://github.com/aurora-is-near/lighthouse.git", rev = "b624c3f0d3c5bc9ea46faa14c9cb2d90ee1e1dec" }
 tree_hash = { git = "https://github.com/aurora-is-near/lighthouse.git",  rev = "b624c3f0d3c5bc9ea46faa14c9cb2d90ee1e1dec" }
 merkle_proof = { git = "https://github.com/aurora-is-near/lighthouse.git", rev = "b624c3f0d3c5bc9ea46faa14c9cb2d90ee1e1dec" }
+eth2_hashing = { git = "https://github.com/aurora-is-near/lighthouse.git", rev = "b624c3f0d3c5bc9ea46faa14c9cb2d90ee1e1dec" }
+eth2_ssz = { git = "https://github.com/aurora-is-near/lighthouse.git", rev = "b624c3f0d3c5bc9ea46faa14c9cb2d90ee1e1dec" }
+eth-types = { path = "../../contracts/near/eth-types/", features = ["eip1559"]}
+contract_wrapper = { path = "../contract_wrapper" }
+eth2-utility  = { path = "../../contracts/near/eth2-utility" }
+finality-update-verify = { path = "../finality-update-verify" }
+log = { version = "0.4", features = ["std", "serde"] }
```

If we are changing dependency order, it also makes sense to split to some "groups"

```suggestion
eth2_hashing = { git = "https://github.com/aurora-is-near/lighthouse.git", rev = "b624c3f0d3c5bc9ea46faa14c9cb2d90ee1e1dec" }
eth2_ssz = { git = "https://github.com/aurora-is-near/lighthouse.git", rev = "b624c3f0d3c5bc9ea46faa14c9cb2d90ee1e1dec" }

eth-types = { path = "../../contracts/near/eth-types/", features = ["eip1559"]}
eth2-utility  = { path = "../../contracts/near/eth2-utility" }

contract_wrapper = { path = "../contract_wrapper" }
finality-update-verify = { path = "../finality-update-verify" }

log = { version = "0.4", features = ["std", "serde"] }
```